frontend/src/lib/components/LemonBubble/LemonBubble.scss

```diff
@@ -0,0 +1,93 @@
+@import '~/vars';
+
+$lemon-counter-small: 1.2rem;
+$lemon-counter-font-small: 0.6rem;
+$sf-small: 0.8;
+$sf-large: 1.2;
+$position-offset: 0.5rem;
```

Comment:
Let's use CSS vars whenever possible (i.e. if we can avoid using Sass functions, which is almost always):
- no dependency on a preprocessor
- inheritance working just like with regular inherited CSS properties
- there's ALWAYS a single source of truth, even when working with a fully built frontend – no need to switch back and forth between the browser and editor, just edit the relevant `--variable` in devtools